src/lib/query.test.js

```diff
@@ -185,6 +186,62 @@ describe("queryCommitCount", () => {
   });
 });
 
+describe("queryRenovatePRs", () => {
+  test("returns renovate prs if the requests succeeds", async () => {
+    const response = {
+      status: 200,
+      data: [
+        {
+          id: 1474610853,
+          number: 1,
+          title:
+            "chore(deps): update js-devtools/npm-publish digest to e42e372 - autoclosed",
+          created_at: "2022-12-04T08:47:37Z",
+          updated_at: "2022-12-07T17:08:04Z",
+          closed_at: "2022-12-07T17:08:01Z",
+          pull_request: {
+            html_url: "https://www.github.com/owner/repo/pull/1",
+          },
+        },
```

Comment:
Could you add one more test object with different values to just confirm it's parsing out the values as expected.

Reply:
done